Events/OnReactionAdd(ReactionSystem)

```diff
@@ -106,19 +106,121 @@
       "name": "End Action Sequence"
     },
     {
-      "tosend": "Found Right Reaction",
-      "color": "#00ff80",
-      "name": "Send Message to Console"
+      "comment": "🔔 Thunder Role 🔔",
+      "color": "#ffff00",
+      "name": "Comment"
     },
     {
-      "tosend": "Found Right Reaction",
-      "color": "#00ff80",
-      "name": "Send Message to Console"
+      "info": "0",
+      "find": "866361823742001153",
+      "storage": "1",
+      "varName": "thunder_role",
+      "name": "Find Role"
     },
     {
-      "tosend": "Found Right Reaction",
-      "color": "#00ff80",
-      "name": "Send Message to Console"
+      "reaction": "1",
+      "varName": "reactionadd",
+      "info": "7",
+      "storage": "1",
+      "varName2": "member",
+      "name": "Store Reaction Info"
+    },
+    {
+      "member": "2",
+      "varName": "member",
+      "role": "3",
+      "varName2": "thunder_role",
+      "iftrue": "0",
+      "iftrueVal": "23",
+      "iffalse": "3",
+      "iffalseVal": "5",
+      "name": "Check If Member has Role MOD"
```

Comment:
If the member has the role, it still thinks that "mem" doesn't have role, so it tries to remove it.

Reply:
This has with the event of the mod, that can find the correct variable, when looking, trying to make it look for another one doesn't work either!